src/main/java/uk/gov/hmcts/reform/sscs/config/HmcHearingsEventTopicListener.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.reform.sscs.config;
+
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusErrorContext;
+import com.azure.messaging.servicebus.ServiceBusException;
+import com.azure.messaging.servicebus.ServiceBusFailureReason;
+import com.azure.messaging.servicebus.ServiceBusProcessorClient;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessage;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessageContext;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.sscs.model.HmcMessage;
+
+import java.util.Objects;
+import java.util.concurrent.CountDownLatch;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+@Slf4j
+@Data
+@Configuration
+public class HmcHearingsEventTopicListener {
+
+    @Value("${azure.service-bus.connectionString}")
```

Comment:
Could you please make the property name more descriptive? We will be listening to multiple queues.